doc/source/serve/index.md

```diff
@@ -127,6 +127,100 @@ Because it's built on top of Ray, you can run it anywhere Ray can: on your lapto
 Check out {ref}`getting-started` and {doc}`key-concepts`, look at the {ref}`serve-faq`,
 or head over to the {doc}`tutorials/index` to get started building your Ray Serve applications.
 
+
+```{eval-rst}
+.. panels::
+    :container: text-center
+    :column: col-lg-6 px-2 py-2
+    :card:
+
+    **Getting Started**
+    ^^^
+
+    Start with our quick start tutorials for :ref:`deploying a single model locally<getting-started>`, how to :ref:`convert an existing model into a Ray Serve deployment<converting-to-ray-serve-deployment>` .
+    These concrete examples will give you an idea of how to use Ray Serve.
+
+    +++
+    .. link-button:: getting-started
+        :type: ref
+        :text: Get Started with Ray Serve
+        :classes: btn-outline-info btn-block
+    ---
+
+    **Key Concepts**
+    ^^^
+
+    Understand the key concepts behind Ray Serve.
+    Learn about :ref:`Deployment<serve-key-concepts-deployment>`, :ref:`how to query it<serve-key-concepts-query-deployment>` and the :ref:`Deployment Graph<serve-key-concepts-deployment-graph>` API for composing models into a graph structure.
+
+    +++
+    .. link-button:: key-concepts
+        :type: ref
+        :text: Learn Key Concepts
+        :classes: btn-outline-info btn-block
+    ---
+
+    **User Guide**
+    ^^^
+
+    Learn how to :ref:`manage deployments<managing-deployments>`, :ref:`handle dependencies<handling-dependencies>`, :ref:`call deployments via HTTP<serve-http>`, :ref:`call deployments from Python<serve-handle-explainer>`, :ref:`deploy, update, and monitor your application with Ray Serve<serve-deploy-tutorial>`, :ref:`serve multiple ML models<serve-ml-models>`, :ref:`debug and monitor your Serve applications<serve-monitoring>`.
```

Comment:
```suggestion
    Learn how to :ref:`manage deployments<managing-deployments>`, :ref:`handle dependencies<handling-dependencies>`, :ref:`call deployments via HTTP<serve-http>`, :ref:`call deployments from Python<serve-handle-explainer>`, :ref:`deploy, update, and monitor your application with Ray Serve<serve-deploy-tutorial>`, :ref:`serve multiple ML models<serve-ml-models>`, and :ref:`debug and monitor your Serve applications<serve-monitoring>`.
```

Reply:
This is very long, I don't think we need to laundry-list every user guide. How about a more generic/concise description like "Learn best practices for common patterns like handling dependencies and calling deployments via HTTP and Python."